shared/rollouts/features.py

```diff
@@ -0,0 +1,3 @@
+PARALLEL_UPLOAD_PROCESSING_BY_ORG = Feature(
```

Comment:
What's this?

Reply:
i wanted a "real" example feature to serve as a reference and placeholder, but for that actual experiment i'll probably put the feature definition in `worker`. i'll comment it out / explain in that file